src/Sulu/Bundle/TagBundle/Tag/TagManager.php

```diff
@@ -122,6 +139,17 @@ public function save($data, $id = null)
 
             $this->em->flush();
 
+            $this->domainEventDispatcher->dispatch(new DomainEvent(
```

Comment:
we cannot use the `DoctrineDomainEventCollector` here because we know the `id` of a created tag only after it was flushed to the database. not really happy that we need to inject the `DomainEventDispatcher` and the `DoctrineDomainEventCollector` because of this.